tv_apps/smart-system/fxa/test/marionette/test-fxa-client/js/fxa_iac_client.js

```diff
@@ -1 +0,0 @@
-../../../../../../../shared/js/fxa_iac_client.js
```

Comment:
It seemed to be a file permission change. I don’t kwow where it came from, but I’ve reverted it.